api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectAuthenticationSaslPlain.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import io.strimzi.crdgenerator.annotations.Description;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Configures the Kafka Connect authentication
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class KafkaConnectAuthenticationSaslPlain extends KafkaConnectAuthentication {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_SASL_PLAIN = "sasl-plain";
```

Comment:
I concerned a little bit about `plain` and wondered if it could be confused with `PLAINTEXT` protocol, especially when we configure it from client side without much context. I am totally agree with you looking at the code repo "plain" is more consistent with `scram-sha-512`,
I will make the changes to all related types, class names and variable names.

Reply:
code updated, please review ..